[FEATURE]: add information about updating jayvee extension in the docs (#480)
jvalue/jayvee: apps/docs/docs/user/intro/intro.md

---
sidebar_position: 1
title: Getting Started
---

# Introduction to Jayvee

Jayvee is a domain-specific language (DSL) for automated processing of data pipelines.
The Jayvee interpreter allows executing such data pipelines on local machines.

> Data engineers can use Jayvee and its interpreter to clean and preprocess data for later activities like data science or machine learning.

Reviewer: What are these changes about? If they are from you, I'd revert them just to keep this focused.

Author: These changes were conflicts with the main branch and needed to be resolved.

> Updating the interpreter is done by reinstalling it using `npm`. Make sure to also update the [VSCode plugin](#vscode-plugin) to match the installed interpreter if you use it.
> 
> IMPORTANT: If a new Jayvee update is available, you need to update your VSCode extension for Jayvee as well.
> 
> More details on updating can be found [here](./update.md).
> 
> ```bash
> npm install -g @jvalue/jayvee-interpreter
> ```

Reviewer: The point of this was to move all the update docs to one place, not keep it around in two places. So here, please just link to the update page and remove the rest. E.g.

```suggestion
Details about how to update Jayvee and the VSCode extension can be found [here](./update.md).
```
